web/templates/layout/app.html.eex

```diff
@@ -22,19 +22,19 @@
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <%= link "Pairmotron", to: page_path(@conn, :index), class: "logo navbar-brand" %>
+          <%= link "Pairmotron", to: pair_path(@conn, :index), class: "logo navbar-brand" %>
         </div>
         <div class="collapse navbar-collapse" id="navbar-collapse-1">
           <ul class="nav navbar-nav navbar-right">
-            <li><%= link "Pairs", to: page_path(@conn, :index) %></li>
+            <li><%= link "Pairs", to: pair_path(@conn, :index) %></li>
             <li><%= link "Groups", to: group_path(@conn, :index) %></li>
             <li><%= link "Projects", to: project_path(@conn, :index) %></li>
             <li><%= link "My Retros", to: pair_retro_path(@conn, :index) %></li>
             <li><%= link "Profile", to: profile_path(@conn, :show) %></li>
               <%= case @conn.assigns[:current_user] do %>
                 <% nil -> %>
                   <%= link "Login", class: "btn btn-success navbar-btn", role: "button", to: session_path(@conn, :new) %>
-                <% current_user -> %>
+                <% _current_user -> %>
```

Comment:
Missed this from the navbar change. I should remember to compile fresh before committing.